frontend/src/components/LogIn/login.css

```diff
@@ -0,0 +1,120 @@
+body {
+    background-color: #F4EFFA;
+    padding-top: 10%;
+    padding-bottom: 10%;
+}
+
+h1 {
```

Comment:
so selectors like `h1` and `body` are very generic and would end up causing other `h1`s outside of this page to adopt the styling you have defined here.

This can cause unintended styling issues across multiple pages.

Look into https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/ and try to understand what problem CSS Modules is trying to solve! 

In general, what you'd have is something called a "global style sheet" that your entire application should be styled as and then you'd have module style sheets that dictate how this specific page or component should be styled as.

You can imagine a global style sheet might be used to determine present font sizes, or fonts themselves but your module css sheet might dictate positioning or override the global defaults.